content/en/docs/tutorials/security/apparmor.md

```diff
@@ -122,8 +122,7 @@ gke-test-default-pool-239f5d02-xwux: kubelet is posting ready status. AppArmor e
 
 {{< note >}}
 AppArmor is currently in beta, so options are specified as annotations. Once support graduates to
-general availability, the annotations will be replaced with first-class fields (more details in
-[Upgrade path to GA](#upgrade-path-to-general-availability)).
+general availability (GA), the annotations will be replaced with first-class fields.
```

Comment:
👋 @kadtendulkar . Generally, the docs not include statements about the state of future features. Is it possible to limit the note to the first sentence?

Reply:
What's the overall aim of this PR? It's editing a sentence that, strictly speaking, shouldn't be in the docs.

I'm happy to see a bigger cleanup here.

Reply:
Hello @divya-mohan0209 and @mehabhalodiya , please review PR.

Reply:
> Hello @divya-mohan0209 and @mehabhalodiya , please review PR.

I think we are ok to remove the broken link since it's broken in prod.

@sftim that cleanup might make for a good good-first-issue, wdyt? I'm happy to create an issue for it

Reply:
Hello @shannonxtreme , can we label this issue as a good first issue?

Reply:
Hey @kadtendulkar , I think there are a couple of action items here based on what I've understood from the conversation
- The entire statement "Once support graduates to general availability (GA), the annotations will be replaced with first-class fields." is a prediction for a third-party tool's feature. So we should be eliminating that sentence entirely to keep the document's content aligned with our policy.
- Shannon or you could open a good-first-issue for a clean-up of this specific doc since there might be more of such content.

@sftim / @shannonxtreme , please correct me if I understood it wrong?

Reply:
Hello @shannonxtreme may I create a good first issue for this?